datafusion/functions-aggregate/Cargo.toml

```diff
@@ -40,6 +40,7 @@ path = "src/lib.rs"
 [dependencies]
 ahash = { workspace = true }
 arrow = { workspace = true }
+arrow-array = { workspace = true }
```

Comment:
use arrow if possible like `use arrow::array::Array; or arrow::datatypes::Field` to avoid re-import crate

Reply:
Fixed